app/views/admin/attachments/index.html.erb

```diff
@@ -31,6 +31,11 @@
         <% end %>
       </ul>
 
+      <p class="alert alert-warning">
```

Comment:
Could we make this a `div` and then use a `p` for each sentence inside it to leave a small gap between them?